scripts/package-build/netfilter/package.toml

```diff
@@ -13,3 +13,8 @@ sudo dpkg -i ../libnftnl*.deb
 sudo mk-build-deps --install --tool 'apt-get --yes --no-install-recommends'
 dpkg-buildpackage -uc -us -tc -b
 """
+
+[dependencies]
+packages = [
+    "asciidoc-base"
```

Comment:
It also requires `bison`, could you recheck?

Reply:
https://salsa.debian.org/pkg-netfilter-team/pkg-nftables/-/blob/master/debian/changelog#L63
https://salsa.debian.org/pkg-netfilter-team/pkg-nftables/-/commit/7ace3531e2f0983976cb039383090ded881f854f
https://salsa.debian.org/pkg-netfilter-team/pkg-nftables/-/merge_requests/10

Debian Maintainer removed that dep in 1.0.9-1 which is the same version of current vyos.
After some tracing, netfilter shipes with ".c" files which is the target from ".y"
And there is a file that mentioned, you will need bison if and only if you modified the .y files.

Reply:
hmm, it seems it needs yacc or bison for sure.
I need to check again